ptx/S_12_4_FTCLI.ptx

```diff
@@ -72,30 +56,17 @@
               notice?
               \saveCount
             </p>
-          </li>
-        </ol>
+       </task>
       </p>
 
       <p>
-        We've used Clairaut's Theorem to argue that a vector field in <m>\R^2</m>
-        is not a gradient vector field when <m>\partial F_1/\partial
-        y\neq \partial F_2/\partial x</m>,
-        and earlier in this preview activity,
-        you verified that a given vector field was the gradient of a
-        particular function of two variables.
-        Clairaut's Theorem holds for
-        functions of three variables.
-        However, in that case there are six mixed partials to
-        calculate, and thus it can be rather tedious.
-        The remaining parts of
-        this preview activity suggest a process for determining if a vector
-        field in <m>\R^3</m> is conservative as well as finding a potential
+        We've used Clairaut's Theorem to argue that a vector field in <m>\R^2</m> is not a gradient vector field when <m>\partial F_1/\partial y\neq \partial F_2/\partial x</m>, and earlier in this preview activity, you verified that a given vector field was the gradient of a particular function of two variables. Clairaut's Theorem holds for functions of three variables. However, in that case there are six mixed partials to calculate, and thus it can be rather tedious.
+        The remaining parts of this preview activity suggest a process for determining if a vector field in <m>\R^3</m> is conservative as well as finding a potential
         function for the vector field.
       </p>
 
       <p>
-        Let <m>\vG(x,y,z) = \langle 3e^{y^2}+z\sin(x),6xy e^{y^2} -
-        z,3z^2-y-\cos(x)\rangle</m> and <m>\vH(x,y,z) = \langle 3x^2 y,x^3+2yz^3,xz+3y^2z^2\rangle</m>.
+        Let <m>\vG(x,y,z) = \langle 3e^{y^2}+z\sin(x),6xy e^{y^2} - z,3z^2-y-\cos(x)\rangle</m> and <m>\vH(x,y,z) = \langle 3x^2 y,x^3+2yz^3,xz+3y^2z^2\rangle</m>.
 
         <ol>
           \restoreCount
```

Comment:
This appears to be leftover LaTeX. There's not really a PTX construct for putting text between list items or between tasks of an `activity` or `exploration`. The best I've come up with is to put the `p`'s that were originally between list items into the next `task`.